flash/image/classification/model.py

```diff
@@ -32,7 +32,7 @@
 )
 from flash.image.classification.adapters import TRAINING_STRATEGIES
 from flash.image.classification.backbones import IMAGE_CLASSIFIER_BACKBONES
-from flash.image.classification.heads import IMAGE_CLASSIFIER_HEADS
+from flash.image.classification.head import IMAGE_CLASSIFIER_HEADS
```

Comment:
I think this was a typo? should be `heads`, I guess:

```suggestion
from flash.image.classification.heads import IMAGE_CLASSIFIER_HEADS
```

Reply:
OK this is solved with the latest commit, the name of the file was `head` and not `heads` so the typo was inside the file name not the import